README.md

```diff
@@ -23,6 +23,17 @@ Mac OS X with Homebrew:
 brew install gcc cmake protobuf protobuf-c yajl
 ```
 
+If you don't have `gtest` installed, do it now:
```

Comment:
What kind of compiler/linkeder error did you experience? It should be cloned automatically into the build folder: https://github.com/0x7f/protog/blob/48f97c6d98b12a3a3e17aa2c9325f6de98aa0df0/test/CMakeLists.txt#L8-L19